### ISSUE TITLE
NVDA 2023.3 and alpha: contracted braille input doesn't work

### ISSUE THREAD

**Issue by aaclause** [nvaccess/nvda#15773]
Regression introduced in #15163 / commit 7b31977

### Steps to reproduce:

Set a contracted braille input table and attempt to use it.

### Actual behavior:

The following error occurs and braille display doesn't work properly:

```
error executing script: <bound method GlobalCommands.script_braille_dots of <globalCommands.GlobalCommands object at 0x052FAAF0>> with gesture 'point 6'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 295, in executeScript
  File "globalCommands.pyc", line 3617, in script_braille_dots
  File "brailleInput.pyc", line 262, in input
  File "brailleInput.pyc", line 233, in _reportUntranslated
  File "brailleInput.pyc", line 317, in updateDisplay
AttributeError: 'BrailleHandler' object has no attribute '_doCursorMove'
```

### Expected behavior:

No such error or behavior.

#### NVDA installed/portable/running from source:
Any

#### NVDA version:
2023.3 and alpha-29936,a5d9e5b5

#### Windows version:
N/A

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
